### ISSUE TITLE
Incident in New York, New York - WIP

### ISSUE THREAD

**Issue by emarcey** [2020PB/police-brutality#1398]
## Location

New York, New York

## Date

August 7th

## Description

Officers gathered around protest organizer's house to arrest him. He asked for the warrant, telling police to slide it under the door; police said they had a warrant but were working on it.

After several hours police, stood down and retreated.

The following day, the protester turned himself in, but plans to fight the charge.

He is charged with assault of an officer, after allegedly turning a megaphone on full volume and yelling into an officer's ear.

He states that about 30 cars, officers with K9s, and sharpshooters on the roof were stationed outside his apartment. The NYPD has not confirmed these details.

## Links

https://www.cnn.com/2020/08/08/us/nypd-protest-derrick-ingram-trnd/index.html
https://www.instagram.com/p/CDm98CulaPa/?hl=en
https://reddit.com/r/2020PoliceBrutality/comments/i5idai/nypd_with_riot_gear_and_dogs_attempt_to_enter/
https://i.imgur.com/uYg88aF.png
https://twitter.com/WarriorsITG/status/1291771654272692224
https://twitter.com/WarriorsITG/status/1291759331315257344
https://twitter.com/itsa_talia/status/1291767865985597440
https://twitter.com/itsa_talia/status/1291763679877312512
https://twitter.com/itsa_talia/status/1291775747149959169

**Comment by Murkantilism** [comment]
@emarcey is this issue waiting on any new information? The NY Times [confirms](https://www.nytimes.com/2020/08/07/nyregion/nypd-derrick-ingram-protester.html) the use of K9 officers, one helicopter, tactical gear + shields, and "roughly two dozen vehicles".

The NYPD itself has only confirmed they used facial recognition software to ID Derrick Ingram but I see no reason to wait for them to confirm the rest, the NYT is a reputable enough source.

The only detail missing is the presence of sharpshooters; I just reached out to communications @ warriorsinthegarden.org directly to ask if they still contend the NYPD employed sharpshooters when attempting to arrest Mr. Ingram on August 7th of this year.

I think we can go ahead with an initial Pull Request and either omit the sharpshooters or clearly mark it as unconfirmed, I will put one together with your permission. If you prefer to wait that's fine too.

**Comment by emarcey** [comment]
@Murkantilism I think more so what we were waiting for is an understanding of the charges against the individual and their validity. Have you seen anything regarding his alleged assault of an officer?

**Comment by Murkantilism** [comment]
@emarcey yes, the felony charges against Mr. Ingram were reduced to a misdemeanor assault on orders of DA Cyrus Vance, [source 1](https://www.dailymail.co.uk/news/article-8608583/Manhattan-district-attorney-DROPS-felony-charge-against-BLM-leader-accused-assaulting-NYPD-cop.html), [source 2](https://nypost.com/2020/08/08/protester-in-nyc-standoff-with-nypd-turns-himself-in/). Mr. Ingram was released without bail August 8th, 2020. The misdemeanor case is still pending, no updates I could find.

Mr. Ingram's organization, among others, [alleged in mid-August](https://www.theguardian.com/world/2020/aug/16/felony-charges-blm-protesters-suppression-tactic) that over-charging is being used as a suppression tactic.

**Comment by emarcey** [comment]
@Murkantilism I was talking to @SiP-C about this and we were both leaning towards closing this without a PR for two reasons:
1. It's tangentially protest related, but it's more about an assault charge. 
1. Yeah it's definitely over-policing, but the circumstances are very muddy and it doesn't (imo) quite fit the evident police brutality for which we're aiming. Even though the org talks about over-policing, Ingram's charges were dropped to misdemeanor and he was released without bail.

Does that sound okay to you?

**Comment by Murkantilism** [comment]
@emarcey @SiP-C hmm that's a tough one, but I think it is fair to strictly scope this repo to brutality _at_ or _during_ protests.

While I do think this attempted arrest qualifies as police brutality, it's a slippery slope to include it in this repo specifically - we'd have to consider if we'd want to include incidents against "regular" protestors outside of protests, or only protest leaders/organizers. Clearer & simpler to scope to protests specifically.

**Comment by emarcey** [comment]
@Murkantilism. @SiP-C pointed me toward similar events in DC that are currently unfolding: https://twitter.com/ChuckModi1/status/1304068512412033024

Maybe it is worthwhile to expand the scope to excessive arrests outside protests. I agree these are really tricky situations, but we do already have one incident (seattle or pdx) where a woman was beaten and arrest the day after and her child was just like... left in the car. So maybe this pattern is enough to consider adding?

**Comment by emarcey** [comment]
@Murkantilism DC issue: https://github.com/2020PB/police-brutality/issues/1650

Seattle issue I referenced: https://github.com/2020PB/police-brutality/blob/master/reports/Washington.md#protester-targeted-days-after-protest--june-5th

@SiP-C how about the three of us take a vote on this comment for expanding the scope to include these incidents? Thumbs up or down

**Comment by Murkantilism** [comment]
@emarcey I'm downvoting for now because the criteria seem too loosely scoped as is, but I have to think about it more.

If we include Laura Eberhard's arrest (the Seattle mom) I don't really see any criteria that would prohibit including Michael Forest Reinoehl's [arrest too](https://www.nytimes.com/2020/09/03/us/michael-reinoehl-arrest-portland-shooting.html), the Portland man killed when officers tried to arrest him for the shooting of a right-wing activist at a pro-Trump rally 5 days prior.

Like I personally see Michael's arrest is obviously not the same thing, but looking at it objectively, there is little difference - it's unclear if the victim actually committed any crime warranting arrest to begin with (Laura may have [backed her car into 2 bike cops](https://www.buzzfeednews.com/article/davidmack/seattle-police-mother-car-biycles-reverse-arrested)), and even if they had committed a crime the arrests seem heavy-handed, but also seem to have elements of resistance:

- Laura allegedly [assaulted arresting officers](https://twitter.com/SeattlePD/status/1269405735584100352) resulting in the deployment of Oleoresin Capsicum spray (compared to our own incident report that says officers showed up "brandishing" mace from the get-go).
- Michael allegedly pulled a gun out on arresting officers

Compare both of these to Derrick Ingram's arrest, which IMO is a much clearer case of police intimidation in direct retaliation to his activism:
- No warrant to enter his home
- Highly militarized arrest attempt (K9 officers, helicopter support, tactical gear & shields)
- DA condemned police tactics, reduced charges for why he was being arrested in the first place
- Zero resistance from the victim, turned himself in the next day

Though I suppose one could argue that Derrick Ingram was simply lucky to be in his home when the cops showed up, as opposed to his vehicle like Laura. Maybe the arrest would have been swift, he might have physically resisted, and the DA would not have condemned anything and kept the felony charges. That's why I have to think about this more.

**Comment by emarcey** [comment]
Alright that all makes a lot of sense. Let's uh just hold in place for now lol

**Comment by emarcey** [comment]
Closing as out of scope. We can revisit again if need be.
